client/src/templates/Introduction/components/challenges.tsx

```diff
@@ -49,40 +52,72 @@ function Challenges({
 
   const isGridMap = isNewRespCert(superBlock) || isNewJsCert(superBlock);
 
+  const firstIncompleteChallenge = challengesWithCompleted.find(
+    challenge => !challenge.isCompleted
+  );
+
   return isGridMap ? (
-    <ul className={`map-challenges-ul map-challenges-grid `}>
-      {challengesWithCompleted.map((challenge, i) => (
-        <li
-          className={`map-challenge-title map-challenge-title-grid ${
-            isProjectBlock ? 'map-project-wrap' : 'map-challenge-wrap'
-          }`}
-          id={challenge.dashedName}
-          key={`map-challenge ${challenge.fields.slug}`}
-        >
-          {!isProjectBlock ? (
-            <Link
-              onClick={() => handleChallengeClick(challenge.fields.slug)}
-              to={challenge.fields.slug}
-              className={`map-grid-item ${
-                challenge.isCompleted ? 'challenge-completed' : ''
+    <>
+      {firstIncompleteChallenge && (
+        <div className='challenge-jump-link'>
+          <Link
+            className='btn btn-primary'
+            onClick={() =>
+              handleChallengeClick(firstIncompleteChallenge.fields.slug)
+            }
+            to={firstIncompleteChallenge.fields.slug}
+          >
+            {t('buttons.resume-project')}{' '}
+            {blockTitle && <span className='sr-only'>{blockTitle}</span>}
```

Comment:
This is purely an accessibility best practice. The visible label on the button ("Resume Project") should be at the beginning of the accessible name used by screen readers. But since there could potentially be several "Resume Project" buttons available in the DOM at once, I've added the project name itself to the end of the accessible name in the `sr-only` text. So we actually don't want this order changed in translations.

Reply:
Interesting. Sounds good to me.